packages/medusa/src/api/store/product-categories/[id]/route.ts

```diff
@@ -19,11 +19,11 @@ export const GET = async (
     req.remoteQueryConfig.pagination
   )
 
-  if (!category) {
+  if (!category?.length) {
```

Comment:
suggestion: use `refetchEntity` instead of `refetchEntities`?

Reply:
Oh honestly I wasn't aware of it 😄 changing it now